src/PolyhedralGeometry/Polyhedron/standard_constructions.jl

```diff
@@ -866,7 +900,39 @@ julia> n_facets(T)
 12
 ```
 """
-catalan_solid(s::String) = polyhedron(Polymake.polytope.catalan_solid(s))
+function catalan_solid(s::String)
+  if s in _catalan_strings_from_oscar
+    # code used for generation of loaded files can be found at:
+    # https://github.com/dmg-lab/RegularSolidsSrc
+    return getfield(Main, Symbol(s))()
```

Comment:
I forgot to send my comments. There is a different approach to this in https://github.com/oscar-system/Oscar.jl/pull/3739#discussion_r1603000534, what do you think about that? This would circumvent all of this string to function matching as it does the inverse